docs/dev_guide/version_control.rst

```diff
@@ -110,6 +110,14 @@ Commit and push the changes to your GitHub: ::
  git commit -a -m "My first commit"
  git push
 
+In addition to all of this, we support several custom tags you can add anywhere in the commit message.
+Please use these tags extensively, especially for documentation PRs and WIP commits.
+
+*  [skip ci] or [ci skip]  - Skips Appveyor and Travis.
+*  [skip appveyor] - Skips Appveyor only.
+*  [skip travis] or [travis skip] -  These terminate the Travis builds before the install steps, effectively skipping testing.
+*  [docs only] or [build docs] - These terminate all non documentation build jobs in Travis before the install steps.
```

Comment:
Should not be only in one place and linked from the other? Otherwise at some time one will be outdated.

Reply:
Fair, I could not decide where to put it. So I put it everywhere.